os/board/rtl8730e/src/component/bluetooth/example/ble_scatternet/scatternet.c

```diff
@@ -267,15 +267,28 @@ static rtk_bt_evt_cb_ret_t ble_tizenrt_scatternet_gap_app_callback(uint8_t evt_c
     }
 
 #if RTK_BLE_5_0_AE_SCAN_SUPPORT
-		case RTK_BT_LE_GAP_EVT_EXT_SCAN_RES_IND: {
-			rtk_bt_le_ext_scan_res_ind_t *scan_res_ind = (rtk_bt_le_ext_scan_res_ind_t *)param;
-			rtk_bt_le_addr_to_str(&(scan_res_ind->addr), le_addr, sizeof(le_addr));
-			printf("[APP] Ext Scan info, [Device]: %s, AD evt type: 0x%x, RSSI: %i, PHY: 0x%x, TxPower: %d, Len: %d\r\n", 
-					le_addr, scan_res_ind->evt_type, scan_res_ind->rssi,
-					(scan_res_ind->primary_phy << 4) | scan_res_ind->secondary_phy,
-					scan_res_ind->tx_power, scan_res_ind->len);
-			break;
-		}
+    case RTK_BT_LE_GAP_EVT_EXT_SCAN_RES_IND: {
+        rtk_bt_le_ext_scan_res_ind_t *scan_res_ind = (rtk_bt_le_ext_scan_res_ind_t *)param;
+        rtk_bt_le_addr_to_str(&(scan_res_ind->addr), le_addr, sizeof(le_addr));
+        // printf("[APP] Ext Scan info, [Device]: %s, AD evt type: 0x%x, RSSI: %i, PHY: 0x%x, TxPower: %d, Len: %d\r\n", 
+        // 		le_addr, scan_res_ind->evt_type, scan_res_ind->rssi,
+        // 		(scan_res_ind->primary_phy << 4) | scan_res_ind->secondary_phy,
+        // 		scan_res_ind->tx_power, scan_res_ind->len);
+        trble_scanned_device scanned_device;
+        scanned_device.adv_type = scan_res_ind->evt_type;
+        if(scanned_device.adv_type == 0x1b){
+            memcpy(scanned_device.resp_data, scan_res_ind->data, scan_res_ind->len);
+            scanned_device.resp_data_length = scan_res_ind->len;
+        }else if (scanned_device.adv_type == 0x13){
+            memcpy(scanned_device.raw_data, scan_res_ind->data, scan_res_ind->len);
+            scanned_device.raw_data_length = scan_res_ind->len;
+        }
```

Comment:
The current definition only handles values for 0x1b and 0x13. Please modify it so that other values can be passed through the else statement as well.

Reply:
```suggestion
        if(scanned_device.adv_type == 0x1b){
            memcpy(scanned_device.resp_data, scan_res_ind->data, scan_res_ind->len);
            scanned_device.resp_data_length = scan_res_ind->len;
        }else {
            memcpy(scanned_device.raw_data, scan_res_ind->data, scan_res_ind->len);
            scanned_device.raw_data_length = scan_res_ind->len;
        }
```